client/ayon_core/cli_commands.py

```diff
@@ -111,7 +114,7 @@ def publish(paths, targets=None, gui=False):
                 app_full_name,
                 launch_type=LaunchTypes.farm_publish,
             )
-            os.environ.update(env)
+            os.environ |= env
```

Comment:
Do we really feel this is more readable? I prefer the old style myself.

My first thought here was - what does this do?

Reply:
union operators are the future! 🚀

Reply:
Hah, they have really been there for a long time of course ;) I'm just debating there readability.